apps/web-client/src/routes/orders/+layout.svelte

```diff
@@ -0,0 +1,219 @@
+		label: string;
+		href: string;
+		icon: any;
+		// This is used purely for testing purposes
+		linkto?: WebClientView;
+	}
+
+	$: ({ nav: tNav } = $LL);
+
+	let links: Link[];
+	$: links = [
+		{
+			label: tNav.search(),
+			href: appPath("stock"),
+			icon: Search,
+			linkto: "stock"
+		},
+		{
+			label: tNav.inventory(),
+			href: appPath("inventory"),
+			icon: Library,
+			linkto: "inventory"
+		},
+		{
+			label: tNav.outbound(),
+			href: appPath("outbound"),
+			icon: PackageMinus,
+			linkto: "outbound"
+		},
+		{
+			label: tNav.settings(),
+			href: appPath("settings"),
+			icon: Settings,
+			linkto: "settings"
+		},
+		{
+			label: tNav.history(),
+			href: appPath("history/date"),
+			icon: Book,
+			linkto: "history/date"
+		},
+		{
+			label: "Customers",
+			href: appPath("customers"),
+			icon: PersonStanding,
+			linkto: "orders/customers"
+		},
+		{
+			label: tNav.supplier_orders(),
+			href: appPath("supplier_orders"),
+			icon: Truck,
+			linkto: "orders/suppliers"
+		}
+	];
+
+	export let view: WebClientView;
+	export let loaded: boolean;
+
+	const {
+		elements: { trigger, overlay, content, close: closeMobileMenu, portalled },
+		states: { open: mobilMenuOpen }
+	} = createDialog({
+		forceVisible: true
+	});
+
+	/**
+	 * Handle create note is an `on:click` handler used to create a new outbound note
+	 * _(and navigate to the newly created note page)_.
+	 */
+	const handleCreateNote = async () => {
+		const note = await getDB().db?.warehouse().note().create();
+		await goto(appPath("outbound", note.id));
+	};
+</script>
+
+<div data-view={view} data-loaded={loaded} class="flex h-screen w-screen overflow-hidden">
+	<!-- Sidenav -->
+	<div class="flex">
+		<div class="hidden h-screen bg-gray-800 sm:inline-block">
+			<div class="block px-6 py-4">
+				<BookCopy color="white" strokeWidth={2} size={36} />
+			</div>
+
+			<nav class="px-3" aria-label="Main navigation">
+				<ul class="flex flex-col items-center gap-y-3">
+					{#each links as { label, icon, href, linkto }}
+						<TooltipWrapper
+							options={{
+								positioning: {
+									placement: "right"
+								},
+								openDelay: 0,
+								closeDelay: 0,
+								closeOnPointerDown: true,
+								forceVisible: true
+							}}
+							let:trigger
+						>
+							<li {...trigger} use:trigger.action>
+								<a
+									data-linkto={linkto}
+									{href}
+									class="inline-block rounded-sm p-4 text-gray-400 {$page.url.pathname.startsWith(href)
+										? 'bg-gray-900'
+										: 'hover:bg-gray-700'}"
+								>
+									<svelte:component this={icon} size={24} />
+								</a>
+							</li>
+
+							<p slot="tooltip-content" class="px-4 py-1 text-white">{label}</p>
+						</TooltipWrapper>
+					{/each}
+				</ul>
+			</nav>
+		</div>
+	</div>
+	<!-- Sidenav end -->
+
+	<!-- Main content -->
+	<div id="content" data-loaded={loaded} class="relative flex h-screen w-full flex-col overflow-hidden bg-gray-50">
+		<!-- Main section -->
```

Comment:
h-screen should be removed here. The parent div already sets it. Use h-full instead